### ISSUE TITLE
Accessibility: Card tag links color contrast

### ISSUE THREAD

**Issue by gerardo-rodriguez** [cloudfour/pwastats#51]
## Summary

The card tag link color contrast ratio is too low.

![screen shot 2017-04-05 at 2 55 07 pm](https://cloud.githubusercontent.com/assets/459757/24728715/eeeef7a4-1a0f-11e7-95a1-4527041f44f2.png)

## Details

If you run the Chrome Dev Tools Accessibility Audit, you will get this message for all of the card tag links:
> [Warning] Text elements should have a reasonable contrast ratio (45)

If you inspect the element and view the "Accessibility Properties", there are two color suggestions made that provide a higher contrast ratio, `#526da5` and `#1c5096`.
```
⚠️ Contrast ratio: 3.69 
   AA level (4.51): #526da5/#ecf1f4 
   AAA level (7.00): #1c5096/#ecf1f4
```
More info: https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_color_01

**Comment by tylersticka** [comment]
Possibly a better issue for [`cloudfour.com-patterns`](https://github.com/cloudfour/cloudfour.com-patterns) since the `.Card` component is part of that toolkit?

**Comment by gerardo-rodriguez** [comment]
Good point @tylersticka. New issue: https://github.com/cloudfour/cloudfour.com-patterns/issues/421

Closing this one.
